libs/utils/package.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "@libs/utils",
+  "version": "0.0.0",
```

Example description to add; either a description or a README might be helpful for contextualizing what the package should be used for and what functionality can be added in the future.

```suggestion
  "version": "0.0.0",
  "description": "a collection of miscellaneous utilities used by peterportal-api request handlers"
```